pkg/webmiddleware/log_test.go

```diff
@@ -33,83 +33,144 @@ func (ch logEntryChannel) HandleLog(e log.Entry) error {
 	return nil
 }
 
-func (ch logEntryChannel) Expect(t *testing.T, f func(log.Entry)) {
+func (ch logEntryChannel) Expect(t *testing.T, a *assertions.Assertion, f func(*assertions.Assertion, log.Entry)) {
 	select {
 	case e := <-ch:
-		f(e)
+		f(a, e)
```

Comment:
I think it's better convention to use `assertions.New(t)` when you need them, and pass `*testing.T` around